ParticleSystem-iOS/GPHParticleView.m

```diff
@@ -26,6 +29,8 @@ -(id)initWithFrame:(CGRect)frame
   self.layer.needsDisplayOnBoundsChange = YES;
   self.clipsToBounds = NO;
   self.emitterCells = @[];
+  self.emitterBeginTimeOffset = 0;
+  self.emitterUseCurrentMediaTime = false;
```

Comment:
Delete. While we could initialize `beginTimeOffset` and `useCurrentMediaTime`, we don't need to since they are automatically initialized to zero.